newIDE/app/src/MainFrame/index.js

```diff
@@ -2902,6 +2906,9 @@ const MainFrame = (props: Props) => {
             }
             eventsFunctionsExtensionsError={eventsFunctionsExtensionsError}
             onReloadEventsFunctionsExtensions={() => {
+              if (isProjectClosed) {
+                return;
+              }
```

Comment:
This is equivalent to what `reloadProjectEventsFunctionsExtensions` is doing if the project passed is null.
We should prefer this because typing is stronger than "yet another boolean state".